source/include/actbl2.h

```diff
@@ -1418,6 +1419,21 @@ typedef struct acpi_mtmr_entry
 
 } ACPI_MTMR_ENTRY;
 
+/*******************************************************************************
+ *
+ * SDEI - Software Delegated Exception Interface Descriptor Table
+ *
+ * Conforms to "Software Delegated Exception Interface (SDEI)" ARM DEN0054A,
+ * May 8th, 2017. Copyright 2017 ARM Ltd.
```

Comment:
Just out of curiosity, why is there a copyright notice here but not in this commit?
https://github.com/acpica/acpica/pull/269/commits/d00a4eb86e64bb4fa70f57ab5e5ca0a4ca2ad8ef